odata-protocol/11 Data Service Requests.md

```diff
@@ -255,22 +255,24 @@ annotation (see [OData-VocCore](#ODataVocCore)) if this
 annotation is present. If not annotated, the format cannot be predicted
 by the client.
 
-The default format for `Edm.Geo` types is `text/plain` using the WKT
+The default format for `Edm.Geo` types is `text/plain` without `charset` parameter, using the WKT
 (well-known text) format, see rules `fullCollectionLiteral`,
 `fullLineStringLiteral`, `fullMultiPointLiteral`,
 `fullMultiLineStringLiteral`, `fullMultiPolygonLiteral`,
 `fullPointLiteral`, and `fullPolygonLiteral` in
 [OData-ABNF](#ODataABNF).
 
 The default format for single primitive values except `Edm.Binary` and
-the `Edm.Geo` types is `text/plain`. Responses of type
-`Edm.String` can use the `charset` format parameter to specify the
-character set used for representing the string value. Responses for the
-other primitive types follow the rules `booleanValue`, `byteValue`,
+the `Edm.Geo` types is `text/plain`.
+
+Responses for type `Edm.String` MAY use the `charset` format parameter to specify the
+character set used for representing the string value; omission of the `charset` parameter implies the default character set `US-ASCII`.
+
+Responses for the other primitive types follow the rules `booleanValue`, `byteValue`,
 `dateValue`, `dateTimeOffsetValue`, `decimalValue`, `doubleValue`,
 `durationValue`, `enumValue`, `guidValue`, `int16Value`, `int32Value`,
 `int64Value`, `sbyteValue`, `singleValue`, and `timeOfDayValue` in
-[OData-ABNF](#ODataABNF).
+[OData-ABNF](#ODataABNF); they MUST NOT use the `charset` parameter which implies the default character set `US-ASCII`.
```

Comment:
```suggestion
[OData-ABNF](#ODataABNF); they MUST NOT use the `charset` parameter since they use the default character set `US-ASCII`.
```
Sounded as if "the `charset` parameter implies the default character set `US-ASCII`".

Reply:
https://www.rfc-editor.org/rfc/rfc2046.html#section-4.1.2 states
> The default character set, which must be assumed in the absence of a charset parameter, is US-ASCII.

Reply:
Maybe I should add a reference.